### ISSUE TITLE
Preamble for new mapping causes compiler error

### ISSUE THREAD

**Issue by DavidLRowe** [silnrsi/silconverters#11]
When TECkit mapping editor creates a new file, it places the following two lines at the end of the preamble:
```
Left-hand range = 0000-007F
Right-hand range = 0000-007F
```
This causes compiler errors:
```
unexpected identifier: "Left" at line 8
unexpected identifier: "Right" at line 9
```
until these lines are commented out.

It seems that these are items used by the editor itself and should be commented out along with other items in the preamble.

**Comment by DavidLRowe** [comment]
Just guessing, but it seems that line 524 in TECkit Mapping Editor.cs needs to have a semicolon and three spaces inserted before {14} and before {16}.

**Comment by bobeaton** [comment]
this should be fixed in 5.1.0
